package-lock.json

```diff
@@ -13582,15 +13582,15 @@
       }
     },
     "node_modules/husky": {
-      "version": "8.0.1",
-      "resolved": "https://registry.npmjs.org/husky/-/husky-8.0.1.tgz",
-      "integrity": "sha512-xs7/chUH/CKdOCs7Zy0Aev9e/dKOMZf3K1Az1nar3tzlv0jfqnYtu235bstsWTmXOR0EfINrPa97yy4Lz6RiKw==",
+      "version": "7.0.4",
```

Comment:
I'm a bit concerned about this changing with component work as it could make dependency stuff very confusing and diffs messy.

Do you think it would be possible to update to a Node that is current/ receiving security updates so that we can all standardise on a newer `husky` version? https://nodejs.org/en/blog/release/v12.22.12/

I'm also not sure if this part of the lockfile would change with an `npm install`. Do you know if you ran this, or `update` @tomwpaterson-shape?